erpnext/public/js/setup_wizard.js

```diff
@@ -139,36 +139,10 @@ erpnext.setup.slides_settings = [
 		},
 
 		validate: function () {
-			let me = this;
-			let exist;
-
 			if (!this.validate_fy_dates()) {
 				return false;
 			}
 
-			// Validate bank name
-			if(me.values.bank_account){
-				frappe.call({
-					async: false,
-					method: "erpnext.accounts.doctype.account.chart_of_accounts.chart_of_accounts.validate_bank_account",
```

Comment:
Why this is removed? The code was there because many people enter just "Bank Accounts" in Bank field in Setup Wizard and it raises duplication error.